pom.xml

```diff
@@ -3418,6 +3418,35 @@
                 <version>${reactivestreams.version}</version>
             </dependency>
 
+
+            <!-- In this section we are enforcing versions for transitive runtime dependencies until regular upgrade path is available via the parent library -->
+
+            <!-- com.ibm.icu:icu4j:jar:61.1 < org.antlr:antlr4:jar:4.9.1 < org.apache.tinkerpop:gremlin-core:jar:3.5.2  -->
+            <dependency>
+                <groupId>org.antlr</groupId>
+                <artifactId>antlr4</artifactId>
+                <version>4.9.3</version>
```

Comment:
we've typically used version strings - so good to follow that pattern?

Reply:
The same discussion from above